h2o-genmodel/src/main/java/hex/genmodel/easy/CategoricalEncoder.java

```diff
@@ -0,0 +1,11 @@
+package hex.genmodel.easy;
+
+import java.io.Serializable;
+
+public interface CategoricalEncoder extends Serializable {
```

Comment:
User-facing API. I'd always provide Javadoc, even if trivial.

Reply:
I generally like the idea of this interface! A lot!

Reply:
done